### DISCUSSION TITLE
Insert images for container in full height mode.

### DISCUSSION THREAD

**Discussion by AZpercussion** [MonkeyBars3k/ReaScripts#178] [Ideas]
Is it possible to change image mode in notes?
Center/title image looks is not fit enough in some cases. Full height will better, I guess.

![Full height image](https://user-images.githubusercontent.com/70285853/150642997-c2cc3989-a1e6-4829-bf34-a92ea0b56d07.jpg)
.

**Comment by MonkeyBars3k** [comment]
Thanks! Actually full height seems to be the superior choice in general. I'm gonna change it :)
